src/step/update_system_variables_masses.hpp

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+#include "assemble_inertia_matrix_masses.hpp"
+#include "assemble_residual_vector_masses.hpp"
+#include "assemble_stiffness_matrix_masses.hpp"
+#include "step_parameters.hpp"
+
+#include "src/elements/masses/masses.hpp"
+#include "src/math/vector_operations.hpp"
+#include "src/state/state.hpp"
+#include "src/system/calculate_RR0.hpp"
+#include "src/system/calculate_inertial_force.hpp"
+#include "src/system/update_node_state.hpp"
+
+namespace openturbine {
+
+inline void UpdateSystemVariablesMasses(
```

Comment:
I do think the Kokkos stuff is off - it probably works on CPU just fine because it degrades to simple for loops with a single iteration.  The code should look quite a bit different for operating on one node/qp per element, since the beams code is structured to make sense for the case of high order elements with many QPs.  

I think the local views are similar to our `InterpolateToQuadraturePoints` kernel for beams, but it's just a deep copy here because there's only one quadrature point.  I would make them members of masses, at least.